src/dwc_checklist.Rmd

```diff
@@ -250,10 +250,17 @@ vernacular_names %<>% mutate(language = recode(language,
 
 ## Post-processing:
 
-Remove the original columns:
```

Comment:
Hmm, does removing using `raw_colnames` generate an error? If not, I don't see a problem using that list... even if some of its columns have already been removed.